tasks/images.js

```diff
@@ -0,0 +1,31 @@
+/* jshint node: true */
+'use strict';
+
+var _ = require('lodash'),
+    imagemin = require('gulp-imagemin'),
+    rename = require('gulp-rename'),
+    sourcemaps = require('gulp-sourcemaps');
```

Comment:
@luke-denton can you please remove `sourcemaps` - they're won't do anything here.